docs/ext/commands/commands.rst

```diff
@@ -714,7 +714,7 @@ For example, augmenting the example above:
     @commands.command()
     async def ban(ctx, *, flags: BanFlags):
         for member in flags.members:
-            await member.ban(reason=flags.reason, delete_message_days=flags.days)
+            await member.ban(reason=flags.reason, clean_history_duration=flags.days * 86400)
```

Comment:
this makes more sense: 
```
timedelta(days=flags.days)
```

(needs to import timedelta)

Reply:
https://github.com/DisnakeDev/disnake/pull/659/commits/f2a7ffb88cb8ed1e0253602e69be1aad267ac132